tests/test_rechunk.py

```diff
@@ -361,10 +354,14 @@ def test_rechunk_array(
     assert target_array.chunks == tuple(target_chunks_list)
     assert dict(source_array.attrs) == dict(target_array.attrs)
 
+    # print(rechunked.plan[0].dask)
     result = rechunked.execute()
     assert isinstance(result, zarr.Array)
-    a_tar = dsa.from_zarr(target_array)
-    assert dsa.equal(a_tar, 1).all().compute()
+    a_tar = np.asarray(result)
+    print(a_tar.shape, a_tar.dtype)
```

Comment:
(nitpick): left a debug print here. And can / should the `assert` below be uncommented?